packages/headless/src/features/commerce/parameters/parameters-selectors.ts

```diff
@@ -70,6 +75,12 @@ export function activeParametersSelector(
       getSelectedValues,
       'f'
     ),
+    ...getFacets(
```

Comment:
Note to self: we should test that the parameter manager controller correctly extracts excluded facet values

Reply:
Addressed in 7ad3a59d1228d30fec7693d9051a51cde4d9d3b7